src/main/java/com/exasol/parquetio/data/Row.java

```diff
@@ -17,12 +17,20 @@
     public List<Object> getValues();
 
     /**
-     * Returns the value at the given position.
+     * Returns a value at the given position.
      *
-     * @param position position of value in the row
-     * @return Java Object
+     * @param position position of a value in the row
+     * @return Java object
      */
-    public Object getObjectAt(final int position);
+    public Object getValue(final int position);
+
+    /**
+     * Returns a value that corresponds to field name.
+     *
+     * @param fieldName field name of a value in the row
+     * @return Java object
+     */
+    public Object getValue(final String fieldName);
```

Comment:
could you also add a List<String> getFiledNames() that prints all available filed names?

Reply:
Added.